Create Language-Version-History.md

```diff
@@ -0,0 +1,104 @@
+Features Added in F# Language Versions
+====================
+
+# F# 1.0
```

Comment:
Might want to add a link like https://docs.microsoft.com/en-us/archive/blogs/dsyme/welcome-to-dons-f-blog here